var/spack/repos/builtin/packages/py-pyprecice/package.py

```diff
@@ -14,15 +14,18 @@ class PyPyprecice(PythonPackage):
 
     homepage = "https://precice.org"
     git = "https://github.com/precice/python-bindings.git"
-    url = "https://github.com/precice/python-bindings/archive/v2.4.0.0.tar.gz"
+    url = "https://github.com/precice/python-bindings/archive/v3.1.1.tar.gz"
     maintainers("ajaust", "BenjaminRodenberg", "IshaanDesai")
 
-    license("LGPL-3.0")
-
```

Comment:
@BenjaminRodenberg is there a reason for removing the `license()` directive here?

Reply:
Good point. I think there is no reason to remove it and we should keep it. No idea why it was removed in the first place. I'll add it again. Thanks for spotting this.